src/components/map/components/DeparturesDialogSheet.tsx

```diff
@@ -58,28 +70,34 @@ const DeparturesDialogSheet = ({
         <ThemeText
           type="body__secondary"
           color="secondary"
-          style={styles.title}
+          style={[styles.title, styles.paddingHorizontal]}
         >
           {t(DeparturesTexts.header.title)}
         </ThemeText>
-        {!stopDetails.state.isLoading && stopPlace ? (
-          <StopPlaceView
-            stopPlace={stopPlace}
-            showTimeNavigation={false}
-            navigateToDetails={navigateToDetails}
-            navigateToQuay={(quay) => {
-              navigateToQuay(stopPlace, quay);
-            }}
-            isFocused={false}
-            searchTime={searchTime}
-            setSearchTime={setSearchTime}
-            showOnlyFavorites={showOnlyFavorites}
-            setShowOnlyFavorites={setShowOnlyFavorites}
-            testID="departuresContentView"
-            allowFavouriteSelection={false}
-          />
+        {!isSearching && !isLoading ? (
+          stopPlace && (stopPlace.quays?.length ?? 0 > 0) ? (
```

Comment:
I think we can make the logic more readable here. 

1. Update `isLoading` to `const isLoading = state.isLoading || isSearching;`, so you can check just the one variable here.
2. `stopPlace && (stopPlace.quays?.length ?? 0 > 0)` can be simplified to `!!stopPlace?.quays?.length`

Reply:
Fixed!